scripts/build/builders/bouffalolab.py

```diff
@@ -136,20 +136,23 @@ def __init__(self,
         if (enable_ethernet or enable_wifi) and enable_thread:
             raise Exception('Currently, Thread cannot be enabled with Wi-Fi or Ethernet')
 
+        if enable_thread:
+            chip_mdns = "platform"
+        elif enable_ethernet or enable_wifi:
+            chip_mdns = "minimal"
+
         # hardware connectivity support check
         if bouffalo_chip == "bl602":
             if enable_ethernet or enable_thread:
-                raise Exception('SoC %s doesn\'t support connectivity Ethernet/Thread.' % bouffalo_chip)
-
+                raise Exception('SoC {} doesn\'t support connectivity Ethernet/Thread.'.format(bouffalo_chip))
```

Comment:
nit: you could use https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings and use quoting so that single quotes does not need escaping.

```python
raise Exception(f"soc {bouffalo_chip} doesn't support connectivity Ethenet/Thread.")
```

Reply:
Updated